### DISCUSSION TITLE
Passing an exotic component to <Link>?

### DISCUSSION THREAD

**Discussion by pljhll** [konstaui/konsta#182] [Q&A]
In the [Link docs](https://konstaui.com/react/link) there is an example of passing a React Router Link component in the `component` property. However, the docs show that this property only accepts strings.

When I try this in my own project (Typescript) - I get the typing errors that the component isn't a string, which matches the docs.

Is the example in the docs incorrect?

**Comment by Pitouli** [comment]
It technically works to pass a "ReactRouterLink" component, but it indeed raise a Typescript Error
